base/pcre.jl

```diff
@@ -100,6 +101,7 @@ end
 function study(regex::Ptr{Void}, options::Integer)
     # NOTE: options should always be zero in current PCRE
     errstr = Array(Ptr{UInt8},1)
+    errstr[1] = C_NULL
     extra = ccall((:pcre_study, :libpcre), Ptr{Void},
                   (Ptr{Void}, Int32, Ptr{Ptr{UInt8}}),
                   regex, options, errstr)
```

Comment:
It's not exactly a bug. Pcre should zero this field, but I had errors in the way I lowered ccall and this helped me get further